mlflow/models/model.py

```diff
@@ -552,6 +561,11 @@ def log(
             mlflow_model = cls(artifact_path=artifact_path, run_id=run_id, metadata=metadata)
             flavor.save_model(path=local_path, mlflow_model=mlflow_model, **kwargs)
             mlflow.tracking.fluent.log_artifacts(local_path, mlflow_model.artifact_path)
+            tracking_uri = _resolve_tracking_uri()
+            if (
+                tracking_uri == "databricks" or get_uri_scheme(tracking_uri) == "databricks"
+            ) and kwargs["signature"] is None:
```

Comment:
```suggestion
            ) and kwargs.get("signature") is None:
```

Maybe `kwargs` always contains signature, but I'd use `get` here. `kwargs["signature"]` looks unsafe.